### PR TITLE
Cleanup few issues and update variables.tf

### PR DESCRIPTION
**Pull request by vishwabmc**
Restructured the description section in variables.tf, and removed some text which were not applicable.

Signed-off-by: vishwabmc <vishwanath@in.ibm.com>

### CODE REVIEW

File: terraform-hpvs/hello-world/my-settings.auto.tfvars-template [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-zone=3 # Zone within region to create the HPVS into.
-logdna_ingestion_key="Your LogDNA ingestion key"
-logdna_ingestion_hostname="Your LogDNA ingestion hostname" # Example: "syslog-a.eu-gb.logging.cloud.ibm.com"
-# Any other variable you want to set (see variables.tf)
\ No newline at end of file
+ibmcloud_api_key="Your IBM Cloud API key"
+region="ca-tor" # Region to deploy to. Options include eu-gb, jp-tok, br-sao, ca-tor
+zone="2" # Zone within region to create the HPVS into.
+logdna_ingestion_key="Your LogDNA ingestion key". # You can find this in "Linux/ubuntu" section of `Logging sources` tab of "IBM Log Analysis" instance in [cloud.ibm.com](https://cloud.ibm.com)
+logdna_ingestion_hostname="rsyslog endpoint of IBM Log Analysis instance" # Example: "syslog-a.<region>.logging.cloud.ibm.com"
```
Comment (CarstenLeue) [inline terraform-hpvs/hello-world/my-settings.auto.tfvars-template:5]: Note that here `<region>` is not the same as the one on line 2

File: terraform-hpvs/hello-world/my-settings.auto.tfvars-template [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
-region="br-sao" # Region to deploy to. Options include eu-gb, jp-tok, br-sao, ca-tor
-zone=3 # Zone within region to create the HPVS into.
-logdna_ingestion_key="Your LogDNA ingestion key"
-logdna_ingestion_hostname="Your LogDNA ingestion hostname" # Example: "syslog-a.eu-gb.logging.cloud.ibm.com"
-# Any other variable you want to set (see variables.tf)
\ No newline at end of file
+ibmcloud_api_key="Your IBM Cloud API key"
+region="ca-tor" # Region to deploy to. Options include eu-gb, jp-tok, br-sao, ca-tor
+zone="2" # Zone within region to create the HPVS into.
+logdna_ingestion_key="Your LogDNA ingestion key". # You can find this in "Linux/ubuntu" section of `Logging sources` tab of "IBM Log Analysis" instance in [cloud.ibm.com](https://cloud.ibm.com)
```
Comment (CarstenLeue) [inline terraform-hpvs/hello-world/my-settings.auto.tfvars-template:4]: where would I find the `logging sources` tab? I suggest to refer to the `Settings / API Keys` section on the instance instead.

image

File: terraform-hpvs/hello-world/variables.tf [hunk 3]
```diff
@@ -1,36 +1,66 @@
 variable "ibmcloud_api_key" {
-  description = "Enter your IBM Cloud API Key, you can get your IBM Cloud API key using: https://cloud.ibm.com/iam#/apikeys"
+  description = <<-DESC
```
Comment (CarstenLeue) [inline terraform-hpvs/hello-world/variables.tf:2]: I suggest to add `sensitive = true`

File: terraform-hpvs/hello-world/variables.tf [hunk 4]
```diff
@@ -1,36 +1,66 @@
...
+                   https://cloud.ibm.com/iam#/apikeys
+                DESC
 }
 
 variable "region" {
   type        = string
-  description = "Region to deploy to, e.g. eu-gb."
+  description = "Region to deploy to, e.g. eu-gb"
+
+   validation {
```
Comment (CarstenLeue) [inline terraform-hpvs/hello-world/variables.tf:12]: how about the Frankfurt zone (`eu-de`)?

**Review by CarstenLeue — APPROVED** [review]

File: terraform-hpvs/hello-world/variables.tf (hunk 3 above)
Comment (vishwabmc) [reply terraform-hpvs/hello-world/variables.tf:2]: Done

File: terraform-hpvs/hello-world/variables.tf (hunk 4 above)
Comment (vishwabmc) [reply terraform-hpvs/hello-world/variables.tf:12]: HPVS Gen2 is not available in DE. However, I figured, I need to add `us-south`, which I did now and will push an updated PR.

File: terraform-hpvs/hello-world/my-settings.auto.tfvars-template (hunk 2 above)
Comment (vishwabmc) [reply terraform-hpvs/hello-world/my-settings.auto.tfvars-template:4]:
````
logdna_ingestion_key="Your LogDNA ingestion key". # You can find this in "Linux/ubuntu" section of `Logging sources` tab of "IBM Log Analysis" instance in [cloud.ibm.com](https://cloud.ibm.com)
 ```

What you are showing is the Dashboard and not the instance page. I was referring to instance page.

File: terraform-hpvs/hello-world/my-settings.auto.tfvars-template (hunk 1 above)
Comment (vishwabmc) [reply terraform-hpvs/hello-world/my-settings.auto.tfvars-template:5]: Sorry, not following. Could you add some more info ?.  

Line:2 has `region` and line:3 has `zone`

**Review by CarstenLeue — APPROVED** [review]

**Pull request merged**